### ISSUE TITLE
Allow buttons in the GUI toolbar to be customized

### ISSUE THREAD

**Issue by Varriount** [WinMerge/winmerge#1479]
Currently, the GUI buttons present in WinMerge's toolbar is fixed - a user cannot specify which buttons are present, nor what their icons are.
![image](https://user-images.githubusercontent.com/524596/188964971-49b42692-0e31-4799-ba82-b09c7a5bd36f.png)

It would be nice to be able to customize the buttons, so that different workflows could be better supported.

Customizations could include:
  - Enabling/disabling which buttons are displayed.
  - Setting the order and positions of buttons.
  - Setting the icon used for a button.

**Comment by jschof** [comment]
Personally I could do w/o setting the icons if it would help expedite implementation of this very useful feature.

**Comment by yalov** [comment]
I would add all "ignore this, ignore that" from the "options" toolbar item to separate toggles, 
and toggle for show whole files or only the diff, so yes, it would be helpful feature

**Comment by ukyouz** [comment]
I would add Swap Panes to quickly reorder the comparison. It would be super useful.

**Comment by jschof** [comment]
@ukyouz Your request is specifically to add a button to do what the existing 2 approaches already do?

**_Method 1_**

![method 1](https://github.com/WinMerge/winmerge/assets/15448821/8a9e2f8d-0d96-4fca-9951-ecb1cc33084d)


**_Method 2_**

![image](https://github.com/WinMerge/winmerge/assets/15448821/2e3b194f-1b47-4bba-990b-67e4d7612293)

**Comment by ukyouz** [comment]
@jschof Yes! But it is still useful if those buttons can be customized into toolbar.

**Comment by iG8R** [comment]
@jschof 
> Your request is specifically to add a button to do what the existing 2 approaches already do?

Yes, exactly, because it is very inconvenient to rummage through the menu when you can click just 1 button.
This also applies to other functions, such as "Add Synchronization Point".

**Comment by Gav-Brown** [comment]
Hi. I'm new to WinMerge. Solid work and nice tool. I would second the ability to hide icons and thus streamline the toolbar.
